.changes/next-release/Feature-64e06f2b-d633-4423-8397-cce8d26d12cc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Automatically add a new launch configuration when creating SAM applications"
```

Comment:
```suggestion
    "description": "A launch configuration is automatically added to the workspace when creating SAM applications"
```

Reply:
can avoid passive voice (which is generally harder to read)

    "Toolkit automatically adds a launch configuration to the workspace when creating SAM applications"